docker/docker-entrypoint.sh

```diff
@@ -29,14 +29,12 @@ if [ "$1" = 'kibana' ]; then
 		sed -e "s|__PROJECT__|$PROJECT_NAME|" -i /opt/kibana/src/ui/views/chrome.jade
         fi
 
-        if [ "$ELASTICSEARCH_USER" != "" ]; then
+        if [ "$ELASTICSEARCH_USER" != "" -a "$ELASTICSEARCH_PASSWORD" != "" ]; then
                 sed -e "s|^#elasticsearch.username:.*$|elasticsearch.username: \"$ELASTICSEARCH_USER\"|" -i /opt/kibana/config/kibana.yml
                 sed -e "s|^#elasticsearch.password:.*$|elasticsearch.password: \"$ELASTICSEARCH_PASSWORD\"|" -i /opt/kibana/config/kibana.yml
 	else
-                ELASTICSEARCH_USER="kibanaserver"
-                ELASTICSEARCH_PASSWORD="kibanaserver"
-                sed -e "s|^#elasticsearch.username:.*$|elasticsearch.username: \"$ELASTICSEARCH_USER\"|" -i /opt/kibana/config/kibana.yml
-                sed -e "s|^#elasticsearch.password:.*$|elasticsearch.password: \"$ELASTICSEARCH_PASSWORD\"|" -i /opt/kibana/config/kibana.yml
+                echo >&2 'error: missing ELASTICSEARCH_USER or/and ELASTICSEARCH_PASSWORD'
```

Comment:
I think we need a clear message. Something like those environment variables weren't configured and that the user has to configure them before running the container.

By the way, they are env variables, aren't they?

Reply:
Yes, they are. I'm going to change that, thanks